OrleansDashboard/History/TraceHistory.cs

```diff
@@ -96,14 +97,15 @@ public void Add(DateTime now, string siloAddress, SiloGrainTraceEntry[] grainTra
 
             foreach (var historicValue in history.Where(x => x.Grain == grain))
             {
-                var grainMethodKey = $"{grain}.{historicValue.Method}";
```

Comment:
Should it not be faster to precalculate this value in the Add() method?

Reply:
possibly, worth benchmarking :¬)

I found that the DateTime.ToString was _very_ slow.